### PR TITLE
feat(spinner): add check class and ratio’s for circle to checkmark size.

### PR DESCRIPTION
**Pull request by daanishnasir**
Add check className and ratio’s for circle to checkmark size.

### CODE REVIEW

File: scss/components/spinner.scss [hunk 1]
```diff
@@ -115,10 +115,12 @@
      &-progress {
       @include spinner-progress-color($spinner--gray__border-color, $spinner-progress--black__border-color);
 
-      $sizes: 16 20 28 36;
+      $ratios: (16, $icon-check_8, 6px), (20, $icon-check_8, 8px), (28, $icon-check_12, 12px), (36, $icon-check_16, 16px);
```
Comment (bfbiggs) [inline scss/components/spinner.scss]: Let's move this to the spinner settings file and remove any variables there that we aren't using anymore. Also let's use a better name at this point than ratios. Maybe spinner-sizes?

File: scss/components/spinner.scss [hunk 2]
```diff
@@ -161,6 +162,17 @@
               font-size: 14px;
               line-height: 22px;
             }
+            &--check {
+              display: flex;
```
Comment (bfbiggs) [inline scss/components/spinner.scss]: Use @include flex; It's a mixin for center flex.

Comment (daanishnasir) [reply scss/components/spinner.scss]: done

File: scss/components/spinner.scss (hunk 1 above)
Comment (daanishnasir) [reply scss/components/spinner.scss]: done

**Pull request merged**